py/mkrules.mk

```diff
@@ -11,7 +11,7 @@ ifeq ($(MICROPY_PREVIEW_VERSION_2),1)
 CFLAGS += -DMICROPY_PREVIEW_VERSION_2=1
 endif
 
-HELP_BUILD_ERROR ?= "See \033[1;31mhttps://github.com/micropython/micropython/wiki/Build-Troubleshooting\033[0m"
+HELP_BUILD_ERROR ?= "See \033[1;31mAdafruit \"Building CircuitPython\" Learn Guide; Adafruit Discord \#circuitpython-dev\033[0m"
```

Comment:
I think you may be able to use a link syntax in terminals. https://github.com/Alhadis/OSC8-Adoption/

Reply:
I didn't include it just because it was longer. I can put it back.